test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -130,12 +130,13 @@ public void KmeansOnnxConversionTest()
                 separatorChar: '\t');
 
             var pipeline = mlContext.Transforms.Normalize("Features").
-                Append(mlContext.Clustering.Trainers.KMeans(features: "Features", advancedSettings: settings =>
+                Append(mlContext.Clustering.Trainers.KMeans(new Trainers.KMeans.KMeansPlusPlusTrainer.Options
                 {
-                    settings.MaxIterations = 1;
-                    settings.K = 4;
-                    settings.NumThreads = 1;
-                    settings.InitAlgorithm = Trainers.KMeans.KMeansPlusPlusTrainer.InitAlgorithm.Random;
+                    FeatureColumn = "Features",
```

Comment:
>Features" [](start = 37, length = 9)

DefaultColumnNames.Features

Reply:
You should told me what I'm wrong and it should be nameof(BreastCancerFeatureVector.Features). or just "Features" since we use same string above. My bad.

---
In reply to: [248115971](https://github.com/dotnet/machinelearning/pull/2158#discussion_r248115971) [](ancestors = 248115971)